README.md

```
@@ -1,2 +1,32 @@
+- Ads and sale offers shoved right in your face
+- Weird system permissions
+- Requires an account just to export
+- Filled to the brim with laggy bloat and AI features that quite literally nobody asked for
+- Subscription and freemium models
+
+After doing a bunch of research, there is **not a single** completely free alternative for Windows, let alone open source. This is what led me to create the OmniBlocks Screen Recorder.
+
+This is a greenfield project, meaning nothing is functional yet. Here's the plan, though
+
+## Tech Stack
+The project will be built using Electron (I know, it's bulky and whatnot, but I only know React, sorry) and using the (Windows Desktop Duplication API)[https://learn.microsoft.com/en-us/windows/win32/direct3ddxgi/desktop-dup-api] for the highest quality and FPS, and then React for the UI.
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(Windows Desktop Duplication API)